catchpoint-mixin/panels.libsonnet

```diff
@@ -9,6 +9,110 @@ local utils = commonlib.utils;
       local stat = g.panel.stat,
       local barGauge = g.panel.barGauge,
       local fieldOverride = g.panel.table.fieldOverride,
+      local alertList = g.panel.alertList,
+      topAvgLoadTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average total load time by test name',
+          targets=[t.topAvgLoadTimeTestName],
+          description='Average total time it took to load the webpage by test name, helping assess the responsiveness and speed of the monitored webpage.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
 
+      topMaxTotalLoadTime:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top max total load time',
+          targets=[t.topMaxTotalLoadTime],
+          description='Maximum total time it took to load the webpage by test name, indicating the worst-case scenario for webpage loading performance.'
```

Comment:
The queries on the right side are not "by test name". They show the test name and the node name indicating the region so that way users can see the "worst-case scenario" instances